src/extension.js

```diff
@@ -410,17 +319,55 @@ const ServiceIndicator = GObject.registerClass({
         this.settings.run_dispose();
 
         // Destroy the PanelMenu.SystemIndicator actors
-        this._item.destroy();
         this.menu.destroy();
 
-        delete AggregateMenu._gsconnect;
+        delete QuickSettingsMenu._gsconnect;
         super.destroy();
     }
 });
 
+const FeatureIndicator = GObject.registerClass(
+class FeatureIndicator extends QuickSettings.SystemIndicator {
```

Comment:
```suggestion
const ServiceIndicator = GObject.registerClass(
class ServiceIndicator extends QuickSettings.SystemIndicator {
```

Renaming for consistency (ServiceIndicator, ServiceToggle).